config/cmake/modules/FindSUNDIALS.cmake

```diff
@@ -22,8 +22,8 @@ mfem_find_package(SUNDIALS SUNDIALS SUNDIALS_DIR
     "include" nvector/nvector_serial.h "lib" sundials_nvecserial
   ADD_COMPONENT NVector_Parallel
     "include" nvector/nvector_parallel.h "lib" sundials_nvecparallel
-  ADD_COMPONENT NVector_ParHyp
-    "include" nvector/nvector_parhyp.h "lib" sundials_nvecparhyp
+  ADD_COMPONENT NVector_MPIPlusX
+    "include" nvector/nvector_mpiplusx.h "lib" sundials_nvecmpiplusx
```

Comment:
There is really no need to remove the `NVector_ParHyp` component since these are just definitions of components -- the actual requirements are in the `find_package(SUNDIALS ...)` command.

The CUDA NVector component can be added here too.

Reply:
I have added `NVector_ParHyp` back and added the CUDA NVector component.

Reply:
`NVector_ParHyp` was used in the MFEM SUNDIALS interface some time in the past but is no longer used anywhere so I think that can be safely removed.